tests/models/blenderbot/test_tokenization_blenderbot.py

```diff
@@ -50,3 +50,23 @@ def test_3B_tokenization_same_as_parlai(self):
     def test_3B_tokenization_same_as_parlai_rust_tokenizer(self):
         assert self.rust_tokenizer_3b.add_prefix_space
         assert self.rust_tokenizer_3b([" Sam", "Sam"]).input_ids == [[5502, 2], [5502, 2]]
+
+    def test_tokenization_for_chat(self):
```

Comment:
these tests require jinja no?

Reply:
Added `@requires_jinja`! It's a requirement of `torch` so it's almost always installed anyway.